pylint/checkers/format.py

```diff
@@ -529,7 +529,7 @@ def visit_default(self, node: nodes.NodeNG) -> None:
             tolineno = node.tolineno
         assert tolineno, node
         lines: list[str] = []
-        for line in range(line, tolineno + 1):
+        for line in range(line, tolineno + 1):  # noqa: B020
```

Comment:
What's the warning here?

Reply:
'Loop control variable overrides iterable it iterates' (It's by design)

Reply:
Are you sure? Can't `for line` be `for index`?
The iterable value just seems to be a integer that's called `line`.

Reply:
I'd rather not touch this, my goal is to add flake8 bugbear not "fix" all legacy issues (especially when the fix is not trivial).